weg-li/App/Resources/JSON/charges.json

```diff
@@ -1,46 +1,49 @@
-    "Parken verbotswidrig auf einem Gehweg",
-    "Parken in einem verkehrsberuhigten Bereich (Zeichen 325.1, 325.2) verbotswidrig außerhalb der zum Parken gekennzeichneten Flächen",
-    "Parken in einem Fußgängerbereich, der (durch Zeichen 239/242.1, 242.2/250) gesperrt war",
-    "Parken in einem Abstand von weniger als 5 Meter vor einem Fußgängerüberweg",
-    "Parken weniger als 8 Meter vor der Kreuzung/Einmündung, obwohl in Fahrtrichtung rechts neben der Fahrbahn ein Radweg baulich angelegt ist",
-    "Parken weniger als 5 Meter vor/hinter der Kreuzung/Einmündung",
-    "Parken im absolutem Haltverbot (Zeichen 283)",
-    "Parken unzulässig im eingeschränkten Haltverbot (Zeichen 286)",
-    "Parken unzulässig auf einer Sperrfläche (Zeichen 298)",
-    "Parken an einer engen/unübersichtlichen Straßenstelle",
-    "Parken im Bereich einer scharfen Kurve",
-    "Parken unzulässig in der zweiten Reihe",
-    "Parkten näher als 10 Meter vor einem Lichtzeichen",
-    "Parken vor oder in einer amtlich gekennzeichneten Feuerwehrzufahrt",
-    "Parken verbotswidrig im Bereich eines Taxenstandes (Zeichen 229)",
-    "Parken verbotswidrig und verhinderten dadurch die Benutzung gekennzeichneter Parkflächen",
-    "Parken im Bereich einer Grundstückseinfahrt bzw. -ausfahrt",
-    "Parken auf einer schmalen Fahrbahn gegenüber einer Grundstückseinfahrt/Grundstücks-ausfahrt",
-    "Parken vor einer Bordsteinabsenkung",
-    "Parken verbotswidrig auf der linken Fahrbahnseite/dem linken Seitenstreifen",
-    "Parken nicht am rechten Fahrbahnrand",
-    "Parken im Fahrraum von Schienenfahrzeugen",
-    "Parken links von einer Fahrbahnbegrenzung (Zeichen 295)",
-    "Parken in einem Verkehrsbereich, der (durch Zeichen 250/251/253/255/260) gesperrt war",
-    "Parken auf einem durch Richtungspfeile (Zeichen 297) gekennzeichneten Fahrbahnteil",
-    "Parken innerhalb einer Grenzmarkierung (Zeichen 299) für ein Haltverbot",
-    "Parken näher als 10 Meter vor einem Andreaskreuz (Zeichen 201)/Zeichen 205 (Vorfahrt gewähren!)/Zeichen 206 (Halt! Vorfahrt gewähren!) und verdeckten dieses",
-    "Parken innerhalb eines Kreisverkehrs (Zeichen 215)",
-    "Parken in einem Abstand von weniger als 15 Metern von einem Haltestellenschild",
-    "Parken, obwohl zwischen Ihrem Fahrzeug und der Fahrstreifenbegrenzung (Zeichen 295/296) ein Abstand von weniger als 3 Metern verblieb",
-    "Parken innerhalb einer Grenzmarkierung (Zeichen 299) für ein Parkverbot",
-    "Parken bei zulässigem Gehwegparken (Zeichen 315) nicht auf dem Gehweg",
-    "Parkten auf einem Sonderfahrstreifen für Omnibusse des Linienverkehrs (Zeichen 245)",
-    "Parken auf einem gekennzeichneten Behindertenparkplatz",
-    "Parken mit Verbrenner vor Elektroladesäule",
-    "Parken auf einer Grünfläche",
-    "Parken auf einer Baumscheibe",
-    "Sonstiges Parkvergehen (siehe Hinweise)"
-]
+{
+  "0": "Sonstiges Parkvergehen (siehe Hinweise)",
```

Comment:
Was bedeuten die Zahlen? Müssen die irgendwie mit in die Auswahl für die User bzw. mit in der Anzeige übermittelt werden?

Reply:
Das ist die Tatbestandsnummer TBNR, wobei alles kleiner gleich 0 keine offizieller Tatbestand ist, aber im Moment noch über die Webseite aus "kompatibilitätsgründen" genutzt werden kann.

Verwendet werden muss die nicht, ist aber hilfreich für mappings und co.

Reply:
Verstehe ich so das wenn die Reports mal an den Server geschickt werden, sollte man die Nummer auch dranhängen. ?

Reply:
Zum Beispiel. Darüber kann man auch Verlinken in den Bußgeldkatalog etc https://www.weg.li/charges/142103